launcher/java/JavaUtils.cpp

```diff
@@ -177,6 +177,12 @@ QStringList addJavasFromEnv(QList<QString> javas)
     QByteArray env = qgetenv("POLYMC_JAVA_PATHS");
 #if defined(Q_OS_WIN32)
     QList<QString> javaPaths = QString::fromLocal8Bit(env).replace("\\", "/").split(QLatin1String(";"));
+    
+    QByteArray envPath = qgetenv("PATH");
+    QList<QString> javaPathsfromPath = QString::fromLocal8Bit(envPath).replace("\\", "/").split(QLatin1String(";"));
```

Comment:
From the table in [qEnvironmentVariable's docs](https://doc.qt.io/qt-6/qtglobal.html#qEnvironmentVariable), it seems that we should use that instead here, like:
```cpp
    auto envPath = qEnvironmentVariable("PATH");
    auto javaPathsfromPath = envPath.replace("\\", "/").split(QLatin1String(";"));
```

Reply:
I guess MMC didn't use it, because it targeted Qt 5.6 :skull:

Reply:
I will try changing it, but my local builds got some issues. So gotta wait on CI to finish to test if it works correctly.

Reply:
> I guess MMC didn't use it, because it targeted Qt 5.6 💀

*still targets 5.4 actually